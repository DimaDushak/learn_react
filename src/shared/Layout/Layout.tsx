import React from 'react';
import styles from './layout.css';

interface Props {
    children: React.ReactNode;
}

export default function Layout({ children }: Props): JSX.Element {
    return <div className={styles.layout}>{children}</div>;
}
