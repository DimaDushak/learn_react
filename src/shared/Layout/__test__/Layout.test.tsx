import { render } from '@testing-library/react';
import React from 'react';
import Layout from '../Layout';

test('should render Layout (snapshot)', () => {
    const { container } = render(<Layout>Верстка</Layout>);

    expect(container.firstChild).toMatchSnapshot();
});
