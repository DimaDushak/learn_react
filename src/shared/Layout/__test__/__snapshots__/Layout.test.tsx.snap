// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Layout (snapshot) 1`] = `
<div
  class="layout"
>
  Верстка
</div>
`;
